package.json

```diff
@@ -1555,6 +1346,215 @@
             "description": "Search symbols in reference assemblies. It affects features requires symbol searching, such as add imports.",
             "order": 80
           },
+          "dotnet.unitTestDebuggingOptions": {
```

Comment:
moved from the O# option section and renamed, let me know if I should keep the same name

Reply:
I am fine the name if debugger team like this